site-content/source/modules/ROOT/pages/download.adoc

```diff
@@ -18,14 +18,14 @@
 [discrete]
 ==== Download the latest Apache Cassandra 4.0 GA release:
 [discrete]
-====== Released on 2022-08-25
+====== Released on 2022-10-23
```

Comment:
How do we normally update versions on the site? Is this something we just do adhoc when we realize they're stale, or perhaps something we can integrate into our release manager pipeline and collab w/Mick / release managers on?

Reply:
Figured as much. What are implications to these diffs in this PR here?

Reply:
I don't know why this PR is updating those dates, it shouldn't be.

Reply:
I honestly don't know how those snuck into this PR. They weren't part of any of the commits. I need to investigate.

Reply:
I don't know what's going on but I suspect it had something to do with the `rebase` I did to squash the commits. 🤦‍♂️

Reply:
I gave up and submitted a new patch.